backend/src/Installer.ts

```diff
@@ -152,71 +160,123 @@ export default class Installer {
     return name + platform
   }
 
-  binaryPath(admin?: boolean) {
+  get extensionName() {
+    let platform = ''
+    if (environment.isWindows32) platform = '.x86-win.exe'
+    else if (environment.isWindows) platform = '.x86_64-64.exe'
+    else if (environment.isMac) platform = '.x86_64-osx'
+    return platform
+  }
```

Comment:
We shouldn't need any of this stuff related to downloading, as we won't be downloading any more.